templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -5,6 +5,9 @@ require File.join(File.dirname(`node --print "require.resolve('@react-native-com
 require 'json'
 podfile_properties = JSON.parse(File.read(File.join(__dir__, 'Podfile.properties.json'))) rescue {}
 
+# Libraries use this env flag to determine whether to add React-RCTFabric as a dependency.
+ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['newArchEnabled'] ? '1' : '0'
```

Comment:
```suggestion
ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['newArchEnabled'] == 'true' ? '1' : '0'
```

Reply:
Nice catch, without this `'false'` would also be resolved to true. Updated